.github/workflows/conda-package.yml

```diff
@@ -267,9 +267,6 @@ jobs:
           activate-environment: ${{ env.TEST_ENV_NAME }}
           python-version: ${{ matrix.python }}
 
-      - name: Remove defaults channel
```

Comment:
It seems you need to specify `channels: conda-forge,nodefaults` for other jobs: `build_windows` and `cleanup_packages`, otherwise `defaults` channel will be present in the configuration.

Reply:
I guess it's because there is `channels: conda-forge,nodefaults` specified. But in other jobs there is `channels: conda-forge` which results to adding `defaults` channel to the configuration. And so it requires `Remove defaults channel` step there.